lib/ApolloController.ts

```diff
@@ -44,7 +44,7 @@ class ApolloController extends Controller {
     if (e.message.match("authentication")) {
       alert("🪵 Please log in to your server.");
     }
-    error("⛔️ Error:", e);
+    console.error("⛔️ Error:", e);
```

Comment:
Thanks, yea I tried to override the javascript console, but we can delete all that code